marshmallow/marshalling.py

```diff
@@ -190,7 +190,7 @@ def run_validator(
 
     def deserialize(
         self, data, fields_dict, many=False, partial=False,
-        unknown=RAISE, dict_class=dict, index_errors=True, index=None,
+        unknown=RAISE, dict_class=dict, list_class=list, index_errors=True, index=None,
```

Comment:
Not sure about the use case, but `dict_class` is configurable, so...

Reply:
I don't think `list_class` is necessary. The only reason we have `dict_class` is to support the `ordered = True` option (in which case `OrderedDict` is used instead of `dict`).

Reply:
Alright, I'll remove the option.